src/extension/background-script/connectors/nativelnbits.ts

```diff
@@ -0,0 +1,86 @@
+import { Method } from "axios";
+
+import LnBits from "./lnbits";
+import Native from "./Native";
+
+const NativeConnector = Native(LnBits);
+
+type PostMessage = {
+  id: string;
+  method: string;
+  url: string;
+  body?: string;
+  params?: Record<string, unknown>;
+  headers: Record<string, string>;
+};
+
+export default class NativeLndHub extends NativeConnector {
+  _nativeRequest(postMessage: PostMessage) {
```

Comment:
@dylancom so far we haven't had this `_nativeRequest` function in the Native class. why not? can we move it there it is the same in `nativelndhub` I think

Reply:
I think because nativelnd.ts isn't integrating it?